static/src/stylesheets/amp/atoms/_snippets.scss

```diff
@@ -57,6 +57,14 @@
         overflow: visible;
     }
 
+    .explainer-snippet__description {
```

Comment:
@annebyrne @tomrf1 AMP validation has been failing since this PR was merged. This does not pass validation. Did it pass when you ran the tests locally?

Unfortunately it doesn't pass as we have already reached our limit of css allowed in AMP. Is it possible to cut down on the css you have added? We will also have to investigate to see what we could possibly delete.

Right now no live blogs are able to be served in AMP on any platform.

Reply:
Yes it appeared to pass when ran locally, there were only warnings visible in TC builds. I will have a look at what we can delete, @regiskuckaertz maybe we should move some of this into the `atom-library` as the AMP styling is just a duplicate of styling elsewhere.

Reply:
Yeah, the styles are in the library, but the conversion to AMP-specific styles remains an open research topic 😄  An easy first heuristic would be to get rid of the styles that are forbidden by AMP.